routes/changeset-query.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
```

Comment:
For reference, [OSM implementation](https://github.com/openstreetmap/openstreetmap-website/blob/master/app/controllers/changeset_controller.rb#L204)